### ISSUE TITLE
Draggable flat list is eating all touches of in list item's scroll view

### ISSUE THREAD

**Issue by canpoyrazoglu** [computerjazz/react-native-draggable-flatlist#253]
**Describe the bug**
Draggable flat list is eating all touches inside the cell's horizontal scroll view. If I swipe horizontally very fast, it scrolls horizontally for a few pixel then stops. If I replace draggable flat list with regular flat list, my inner scroll view works perfectly.

**To Reproduce**
Have scroll view inside the items of draggable flatlist. 

**Platform & Dependencies**
Please list any  applicable dependencies in addition to those below (react-navigation etc).
 - Platform: iOS 14.3
 - React Native or Expo version: 0.63.3
 - Reanimated version: 1.13.1
 - React Native Gesture Handler version: 1.8.0

**Additional context**
I've tried setting all of these on the draggable flatlist:

```
onStartShouldSetResponder={() => false}
onMoveShouldSetResponder={() => false}
onMoveShouldSetResponderCapture={() => false}
onStartShouldSetResponderCapture={() => false}
onResponderTerminationRequest={() => true}
```

I've also tried changing `activationDistance` but doesn't seem to affect.

**Comment by computerjazz** [comment]
hm yep i didn't consider horizontally scrolling items in a vertical container when building this. It might be as easy as adding an `activeOffsetX: Infinity` or equivalent here: https://github.com/computerjazz/react-native-draggable-flatlist/blob/master/src/index.tsx#L1014

**Comment by computerjazz** [comment]
seems to work ok for me -- what version of `react-native-draggable-flatlist` are you on?

![rndfl-nested-scroll](https://user-images.githubusercontent.com/6730148/105918448-c9f10780-5fe8-11eb-92a1-bc2d2df69006.gif)

**Comment by canpoyrazoglu** [comment]
After investigating a bit more and trying different combinations of views and pan handler options on both `DraggableFlatList` and my own views, I managed to get it working. The idea was to abandon my own scroll view which always conflicted with the draggable flat list, and to use a regular `Animated.View` and implement any horizontal swiping action through a `translateX` animation bound to that particular cell's animated view and listen to and touch events directly on that view. Then I hooked to the scroll begin/end events of the draggable flat list and disabled my own horizontal swipe mechanism whenever the control was on draggable flat list (meaning that we're dragging an item).

It worked.

(I'm on 2.5.1)
